src/website/app/demos/ButtonGroup/examples/uncontrolled.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import Button from '../../../../../library/Button';
+import ButtonGroup from '../../../../../library/ButtonGroup';
+
+export default {
+  id: 'uncontrolled',
+  title: 'Uncontrolled',
+  description: `Create an uncontrolled ButtonGroup by using the
+\`defaultChecked\` prop rather than the \`checked\` prop. Note that the
+\`defaultChecked\` prop may be set on the root (where the value may be a number
+or an array of numbers representing the index or indices of the Button(s) to
+target, based on the zero-indexed collection of its nested Buttons) or on
+individual children (where the value defaults to true and thus need not be
```

Comment:
The "defaults to true" bit is just how JSX works. We could probably simply say "where the value is a boolean".